src/components/Search/StatefulSearch.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { forwardRef, useState } from 'react';
+import isEqual from 'react-fast-compare';
+
+import Search from './Search';
+import type { SearchProps } from './Search.types';
+import type { FilterOption } from 'components/Filter';
+
+export const LABEL = 'Friends';
+export const options = [
```

Comment:
why are these being displayed only for the story showcasing?

Reply:
check latest commit for fix